pkg/nb/types.go

```diff
@@ -393,14 +399,15 @@ type AccountAllowedBuckets struct {
 
 // CreateAccountParams is the params of account_api.create_account()
 type CreateAccountParams struct {
-	Name              string                `json:"name"`
-	Email             string                `json:"email"`
-	HasLogin          bool                  `json:"has_login"`
-	S3Access          bool                  `json:"s3_access"`
-	AllowBucketCreate bool                  `json:"allow_bucket_creation"`
-	AllowedBuckets    AccountAllowedBuckets `json:"allowed_buckets"`
-	DefaultResource   string                `json:"default_resource,omitempty"`
-	BucketClaimOwner  string                `json:"bucket_claim_owner,omitempty"`
+	Name              string					`json:"name"`
```

Comment:
notice the indentation here, you have a mix of spaces and tabs, can be solved by using spaces or tabs exclusively.
also happens in `UpdateAccountS3AccessParams, AccountNsfsConfig` and in these two if statements
`if r.NooBaaAccount.Spec.NsfsAccountConfig != nil` in `pkg/noobaaaccount/reconciler.go`